src/imap/mod.rs

```diff
@@ -1463,24 +1481,49 @@ fn get_folder_meaning_by_name(folder_name: &str) -> FolderMeaning {
         "送信済み",
         "보낸편지함",
     ];
+    let spam_names = vec![
```

Comment:
this should be a constant

Reply:
For performance reasons or style?
Performance is not actually important as this function is only used, like, 5 times during configure.
Concerning style: I liked it better in here but we could move it to top-level if you say it's better (and also the `sent_names` variable then, of course)

Reply:
You can have a constant in here, they don't need to be top level.

Reply:
Great, didn't know